src/TinyContainerBuilder.php

```diff
@@ -129,6 +139,12 @@ public function makePublic(string $service): ContainerBuilderInterface
         return $this;
     }
 
+    public function makeAllPublic(): ContainerBuilderInterface
+    {
+        $this->makeAllServicesPublic = true;
```

Comment:
should we make it a little more "explodey" by first checking that this isn't set and throwing a LogicException if it is?

Reply:
Updated.